ui/lib/sync/addon/components/secrets/page/destinations/create-and-edit.hbs

```diff
@@ -4,10 +4,40 @@
 ~}}
 
 <SyncHeader
-  @title="Create destination"
+  @title="Create destination for {{this.displayName}} "
   @breadcrumbs={{array
     (hash label="Secrets sync" route="secrets.overview")
     (hash label="Select destination" route="secrets.destinations.create")
     (hash label="Create destination")
   }}
-/>
\ No newline at end of file
+/>
+
+<hr class="is-marginless has-background-gray-200" />
+
+<form {{on "submit" (perform this.save)}} class="has-top-margin-l">
+  <MessageError @errorMessage={{this.error}} />
+
+  {{#each @destination.formFields as |attr|}}
+    <FormField @attr={{attr}} @model={{@destination}} @modelValidations={{this.modelValidations}} />
+  {{/each}}
+
+  <hr class="has-background-gray-200 has-top-margin-l" />
+  <Hds::ButtonSet>
+    <Hds::Button
+      @text={{if @model.isNew "Create destination" "Save"}}
+      data-test-save
+      type="submit"
+      disabled={{this.save.isRunning}}
```

Comment:
Not sure this was discussed, but will there be a loading state when `this.save.isRunning`?

Reply:
Ignore my question above - just realized we opted to disable the button

Reply:
During the first phase of the button adoption we were setting the icon property to loading as well as disabling. Should we keep things consistent?

Reply:
Yeah! That's good to know - I wasn't sure what we'd landed on. Also the sync save takes a while, so I think a loading state would be a UX improvement